src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-bar bar--0" data-qa="big">
+      <div
+
+      class="search-bar__icon-big"
+      ></div>
+
+      <input
+
+      type="text"
+
+      class="search-bar__input input--0"
+
+      placeholder="Try &quot;Los Angeles&quot;"
+
+      data-qa="keypress"
+      >
+    </form>
+
+    <form class="search-bar bar--1" data-qa="small">
+      <div
+
+      class="search-bar__icon-small"></div>
+
+      <input
+
+      type="text"
+
+      class="search-bar__input input--1"
+
+      placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
```

Comment:
what is going on with your html formatting? read [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)